### PR TITLE
Improve IDX logo in left sidebar so it shows in light mode

### PR DESCRIPTION
**Pull request by clearlyTHUYDOAN**
## Description

IDX logo shows in dark mode but not light mode.

Screenshot 2024-12-02 at 2 00 52 PM

### CODE REVIEW

**Comment by clearlyTHUYDOAN** [comment]
@delucis Am I missing something?

I expected to use the horizontal SVG here like the one on the home page. 

It's showing up as the tiny one though?

**Comment by sarah11918** [comment]
Is there a problem with the one live in docs now? It has text as far as I can tell. Currently live:

image

Is this not what we want?

**Review by delucis — COMMENTED** [review]
Yeah the logo is fine already in dark mode. All that should need fixing is the path which uses white for its fill being updated to use `currentColor` instead.

File: src/components/LeftSidebar/Sponsors.astro [hunk 1]
```diff
@@ -28,28 +28,14 @@
...
...2.25.66a4.22 4.22 0 0 0-1.52 1.78c-.16.33-.27.68-.35 1.04Zm21.05 9.47a7.26 7.26 0 0 1-6.62-3.77 8.45 8.45 0 0 1-.95-4.03c0-1.5.31-2.83.95-4a7.26 7.26 0 0 1 2.68-2.77 7.52 7.52 0 0 1 3.94-1.03c1.63 0 3 .36 4.12 1.1A6.02 6.02 0 0 1 148.1 25l-2.82 1.15a3.68 3.68 0 0 0-1.5-1.76c-.63-.4-1.39-.6-2.27-.6a4.43 4.43 0 0 0-3.83 2.33c-.4.73-.6 1.6-.6 2.62 0 1 .2 1.87.6 2.62a4.23 4.23 0 0 0 3.83 2.33 4.3 4.3 0 0 0 2.4-.63 4.23 4.23 0 0 0 1.52-1.79l2.76 1.21a7 7 0 0 1-6.62 4.06Zm8.14-15.14h2.56v-4.15h3.1v4.15h3.6v2.7h-3.6v7.06c0 .73.15 1.28.44 1.67.3.38.82.57 1.53.57.34 0 .65-.04.92-.14.27-.12.53-.25.8-.4v3.02a5.93 5.93 0 0 1-2.33.43c-1.36 0-2.44-.4-3.25-1.18-.8-.8-1.21-1.9-1.21-3.31v-7.71h-2.56V21.4Zm19.49 14.68V15.47h3.16v20.61h-3.16Zm7.84 0V15.47h6.6c2.16 0 4.02.44 5.58 1.32a8.92 8.92 0 0 1 3.63 3.63c.86 1.55 1.3 3.34 1.3 5.35 0 2.02-.44 3.8-1.3 5.36a9.1 9.1 0 0 1-3.63 3.66 11.4 11.4 0 0 1-5.59 1.3h-6.59Zm3.17-3h3.34c1.55 0 2.88-.28 4-.86a5.98 5.98 0 0 0 2.53-2.53c.58-1.1.86-2.4.86-3.92 0-1.51-.28-2.82-.86-3.91a5.81 5.81 0 0 0-2.53-2.5 8.38 8.38 0 0 0-4-.9h-3.34V33.1Zm15.63-17.61h3.9l4.57 7.4h.17l4.6-7.4h3.86l-6.41 9.9 6.88 10.71h-3.86l-5.07-8.03h-.17l-5.07 8.03h-3.86l6.88-10.7-6.41-9.91Z"/>
```
Comment (yanthomasdev) [inline src/components/LeftSidebar/Sponsors.astro:37]: That should take care of the issue! Tested on Chrome Dev Tools.
```suggestion
<path fill="currentColor" d="M66.5 36.08V15.47h7.08c1.25 0 2.38.27 3.4.8a5.99 5.99 0 0 1 2.42 2.22c.61.94.92 2.04.92 3.28a6.2 6.2 0 0 1-3.34 5.53 7.37 7.37 0 0 1-3.4.78h-3.91v8H66.5Zm3.17-11h4c.75 0 1.38-.15 1.9-.46a3.3 3.3 0 0 0 1.18-1.23c.29-.52.43-1.06.43-1.62a3.1 3.1 0 0 0-.43-1.58 3.3 3.3 0 0 0-3.08-1.73h-4v6.62Zm13.38 11V21.4h2.93v2.16h.18c.23-.5.55-.94.98-1.33.44-.38.94-.69 1.5-.92a4.76 4.76 0 0 1 2.78-.26c.3.04.55.1.78.2v3.23a7.7 7.7 0 0 0-1.12-.37 4.5 4.5 0 0 0-1.15-.15c-.73 0-1.38.2-1.96.6-.56.39-1 .9-1.32 1.56a4.82 4.82 0 0 0-.5 2.19v7.77h-3.1Zm17.59.46a7.41 7.41 0 0 1-6.7-3.83 8.12 8.12 0 0 1-.99-3.97c0-1.46.33-2.77.98-3.94a7.35 7.35 0 0 1 6.71-3.86c1.5 0 2.82.35 3.97 1.06a7.4 7.4 0 0 1 2.7 2.8c.66 1.17.99 2.48.99 3.94s-.33 2.78-.98 3.97a7.4 7.4 0 0 1-6.68 3.83Zm0-2.85c.8 0 1.55-.2 2.25-.57.69-.4 1.24-.97 1.67-1.7a5.2 5.2 0 0 0 .66-2.68c0-1.04-.22-1.92-.66-2.65a4.31 4.31 0 0 0-1.67-1.7 4.53 4.53 0 0 0-6.22 1.7 5.2 5.2 0 0 0-.64 2.65 4.67 4.67 0 0 0 2.36 4.38 4.7 4.7 0 0 0 2.25.57Zm9.37 8.9a9.4 9.4 0 0 1-1.61-.2v-3a3.34 3.34 0 0 0 1.35.38c.54 0 .92-.18 1.16-.52.23-.33.34-.8.34-1.44V21.4h3.1v16.44c0 1.13-.18 2.04-.57 2.73-.36.7-.87 1.2-1.52 1.53-.66.32-1.4.49-2.25.49Zm2.8-23.35a2.1 2.1 0 0 1-1.5-.6 2.1 2.1 0 0 1-.6-1.5c0-.6.2-1.09.6-1.47.42-.4.92-.6 1.5-.6.59 0 1.09.2 1.5.6.4.38.6.87.6 1.47 0 .57-.2 1.07-.6 1.5-.41.4-.91.6-1.5.6Zm11.97 17.3a7.27 7.27 0 0 1-6.54-3.77 8.28 8.28 0 0 1-.94-4c0-1.4.3-2.7.92-3.89a7.35 7.35 0 0 1 2.56-2.85 6.83 6.83 0 0 1 3.83-1.1c1.52 0 2.8.33 3.86.99a6.35 6.35 0 0 1 2.45 2.67 8.57 8.57 0 0 1 .77 4.81h-11.34c.08.8.28 1.5.6 2.1a4.3 4.3 0 0 0 3.92 2.28c.98 0 1.8-.22 2.45-.66a5.12 5.12 0 0 0 1.58-1.7l2.56 1.24a7.95 7.95 0 0 1-2.59 2.82 7.36 7.36 0 0 1-4.09 1.06Zm-4.26-9.47h8.15a3.37 3.37 0 0 0-.29-1.15 3.7 3.7 0 0 0-1.98-1.99 4.15 4.15 0 0 0-1.76-.34c-.84 0-1.6.22-2.25.66a4.22 4.22 0 0 0-1.52 1.78c-.16.33-.27.68-.35 1.04Zm21.05 9.47a7.26 7.26 0 0 1-6.62-3.77 8.45 8.45 0 0 1-.95-4.03c0-1.5.31-2.83.95-4a7.26 7.26 0 0 1 2.68-2.77 7.52 7.52 0 0 1 3.94-1.03c1.63 0 3 .36 4.12 1.1A6.02 6.02 0 0 1 148.1 25l-2.82 1.15a3.68 3.68 0 0 0-1.5-1.76c-.63-.4-1.39-.6-2.27-.6a4.43 4.43 0 0 0-3.83 2.33c-.4.73-.6 1.6-.6 2.62 0 1 .2 1.87.6 2.62a4.23 4.23 0 0 0 3.83 2.33 4.3 4.3 0 0 0 2.4-.63 4.23 4.23 0 0 0 1.52-1.79l2.76 1.21a7 7 0 0 1-6.62 4.06Zm8.14-15.14h2.56v-4.15h3.1v4.15h3.6v2.7h-3.6v7.06c0 .73.15 1.28.44 1.67.3.38.82.57 1.53.57.34 0 .65-.04.92-.14.27-.12.53-.25.8-.4v3.02a5.93 5.93 0 0 1-2.33.43c-1.36 0-2.44-.4-3.25-1.18-.8-.8-1.21-1.9-1.21-3.31v-7.71h-2.56V21.4Zm19.49 14.68V15.47h3.16v20.61h-3.16Zm7.84 0V15.47h6.6c2.16 0 4.02.44 5.58 1.32a8.92 8.92 0 0 1 3.63 3.63c.86 1.55 1.3 3.34 1.3 5.35 0 2.02-.44 3.8-1.3 5.36a9.1 9.1 0 0 1-3.63 3.66 11.4 11.4 0 0 1-5.59 1.3h-6.59Zm3.17-3h3.34c1.55 0 2.88-.28 4-.86a5.98 5.98 0 0 0 2.53-2.53c.58-1.1.86-2.4.86-3.92 0-1.51-.28-2.82-.86-3.91a5.81 5.81 0 0 0-2.53-2.5 8.38 8.38 0 0 0-4-.9h-3.34V33.1Zm15.63-17.61h3.9l4.57 7.4h.17l4.6-7.4h3.86l-6.41 9.9 6.88 10.71h-3.86l-5.07-8.03h-.17l-5.07 8.03h-3.86l6.88-10.7-6.41-9.91Z"/>
```

**Review by sarah11918 — APPROVED** [review]
Confirming preview is good in both light and dark mode!

**Pull request merged**